### PR TITLE
feat(phrases): add i18n phrases for zh-cn about OICD Authorize page

### PR DESCRIPTION
**Pull request by the-pawn-2017**
I have translated the "authorization page" following third-party IdP login within OIDC into Chinese. I carefully examined the translated result, meticulously cross-checking it using both my personal language preferences and translation software.

### CODE REVIEW

File: packages/phrases-experience/src/locales/zh-cn/description.ts [hunk 1]
```diff
@@ -64,23 +64,23 @@ const description = {
     '你的企业已为电子邮件账户{{email}}启用了单点登录。你可以继续使用以下SSO提供商进行登录。',
   single_sign_on_enabled: '该帐户已启用单点登录',
   /** UNTRANSLATED */
```
Comment (xiaoyijun) [inline packages/phrases-experience/src/locales/zh-cn/description.ts:66]: These flags should be removed since translations are added.

**Review by xiaoyijun — CHANGES_REQUESTED** [review]

File: packages/phrases-experience/src/locales/zh-cn/description.ts (hunk 1 above)
Comment (the-pawn-2017) [reply packages/phrases-experience/src/locales/zh-cn/description.ts:66]: I will fix that, but I find pushing to the master is bad way, so, I will close this, and pulling a new requested

Comment (darcyYe) [reply packages/phrases-experience/src/locales/zh-cn/description.ts:66]: We generally insert a space between full-width characters (such as Chinese, Japanese and Korean characters), and half-width English letters, Arabic numerals, and symbols, to provide a better reading experience.

File: packages/phrases-experience/src/locales/zh-cn/description.ts [hunk 2]
```diff
@@ -64,23 +64,23 @@ const description = {
     '你的企业已为电子邮件账户{{email}}启用了单点登录。你可以继续使用以下SSO提供商进行登录。',
```
Comment (darcyYe) [inline packages/phrases-experience/src/locales/zh-cn/description.ts:64]:
```suggestion
    '你的企业已为电子邮件账户 {{email}} 启用了单点登录。你可以继续使用以下 SSO 提供商进行登录。',
```

File: packages/phrases-experience/src/locales/zh-cn/description.ts [hunk 3]
```diff
@@ -64,23 +64,23 @@ const description = {
...
+  grant_organization_access: '授予组织访问权限:',
   /** UNTRANSLATED */
-  user_scopes: 'Personal user data',
+  user_scopes: '用户个人信息',
   /** UNTRANSLATED */
-  organization_scopes: 'Organization access',
+  organization_scopes: '组织权限',
   /** UNTRANSLATED */
-  authorize_agreement: `By authorize the access, you agree to the {{name}}'s <link></link>.`,
+  authorize_agreement: `你将同意授权给{{name}} <link></link>.`,
```
Comment (darcyYe) [inline packages/phrases-experience/src/locales/zh-cn/description.ts:77]:
```suggestion
  authorize_agreement: `你将同意授权给 {{name}} <link></link>.`,
```

Comment (the-pawn-2017) [reply packages/phrases-experience/src/locales/zh-cn/description.ts:77]: I got it

**Pull request closed without merging**